packages/website/src/pages/leading.js

```diff
@@ -26,6 +26,7 @@ class LeadingPage extends Component {
           <h3>Before the ride</h3>
           <p>Before setting off ensure:</p>
           <ol>
+            <li>No group is larger than ~14 people. If it is, make sure the group splits in to two.</li>
```

Comment:
Wrong place for this discussion but 14 is still a pretty big group!

Reply:
yeah you're probably right, I'll drop it down to 12. At least that way people should feel guilty if it grows to 14.